invoking surround with try catch on:

 Class c;
 c.getConstructor(args);

results in code that also catches SecurityException. This is a run-time 
exception and it is usually bad practice to catch run-time exception. Therefore 
by default only checked exceptions should be added.

invoking surround with try catch on:

 Class c;
 c.getConstructor(args);

results in code that also catches SecurityException. This is a run-time 
exception and it is usually bad practice to catch run-time exception. Therefore 
by default only checked exceptions should be added.
Runtime exception are noew ingnore by Surround with try/catch. Did not add UI 
to control this behaviour since surround with doesn't have a UI at all. And 
offering a global preference doesn't make to much sence.

Fixed in build > I20020416


Surround with try/catch adds run-time exceptions

